Governance: add Conway overview + diagram (#1319)
cardano-foundation/developer-portal: docs/get-started/cardano-cli/governance/governance.md

## Conway era update proposals

> The Conway Era introduces a new governance framework to Cardano, marking a significant evolution in how decisions about the protocol are made. Building on the foundations laid in previous eras, Conway empowers Cardano with a decentralized governance model where decision-making is accessible to stakeholders. Key features include the introduction of *Delegated Representatives (DReps)*, the *Constitutional Committee*, and an decision making role for *Stake Pool Operators* (SPOs). These entities collaborate through a formalized voting system to vote protocol paramter changes that can be proposed by any ada holder. These governance bodies are in charge of treasury funds, adding or removing members from the constitutional committee, making updates to the Cardano Constitution. Conway era makes Cardano more resilient and 100% community-driven.

Reviewer: ```suggestion
The Conway Era introduces a new governance framework to Cardano, marking a significant evolution in how decisions about the protocol are made. Building on the foundations laid in previous eras, Conway empowers Cardano with a decentralized governance model where decision-making is accessible to stakeholders. Key features include the introduction of *Delegated Representatives (DReps)*, the *Constitutional Committee*, and an decision making role for *Stake Pool Operators* (SPOs). These entities collaborate through a formalized voting system to vote on governance actions that can be proposed by any ada holder. These governance bodies are in charge of treasury funds, adding or removing members from the constitutional committee, making updates to the Cardano Constitution and changes to protocol parameters. Conway era makes Cardano more resilient and 100% community-driven.
```

I think this makes more sense

> The process for submitting and voting update proposals undergoes significant changes in the Conway Era. First, Genesis delegations and MIR certificates are eliminated. In `DState`, the fields associated with these features are no longer included, and `DelegEnv` no longer contains the fields it had in the Shelley era, effectively rendering the genesis keys obsolete. 

Reviewer: ```suggestion
The process for submitting and voting protocol update proposals undergoes significant changes in the Conway era. First, Genesis delegations and MIR certificates are eliminated. In `DState`, the fields associated with these features are no longer included, and `DelegEnv` no longer contains the fields it had in the Shelley era, effectively rendering the genesis keys obsolete. 
```

### Governance actions

> In Conway era, any ada holder can submit a Governance action, which is the on-chain method to put a proposal to the consideration of the three governance bodies. There are 7 types of governance actions:

Reviewer: ```suggestion
In the Conway ledger era, any ada holder can submit a Governance action, which is the on-chain method to put a proposal to the consideration of the three governance bodies. There are 7 types of governance actions:
```

| Governance Action | Description |
| ----------------- | ----------- |
| NoConfidence      | A motion to create a state of no-confidence in the current constitutional committee |
| UpdateCommittee   | Changes to the members of the constitutional committee and/or to its signature threshold and/or terms |

> | NewConstitution   | A modification to the off-chain Constitution and the proposal policy script |

Reviewer: ```suggestion
| NewConstitution   | A modification to the off-chain Constitution and the or proposal policy script |
```

Reviewer 2: @Ryun1 can you double-check this suggestion?  Should it b:e _modification to the off-chain Constitution **or** the proposal policy script_?

Reviewer: oops, I meant "and or"
I will edit

Reviewer 2: aha, I didn't read it as the grammatically incorrect but universally accepted `and/or` ... which is logically just an `or` 🤓 (I've corrected the change above)

> | TriggerHF3        | Triggers a non-backwards compatible upgrade of the network; requires a prior software upgrade |

Reviewer: ```suggestion
| TriggerHF         | Triggers a non-backwards compatible upgrade of the network; requires a prior software upgrade |
```

> | ChangePParams     | A change to one or more updatable protocol parameters, excluding changes to major protocol versions ("hard forks") |

Reviewer: ```suggestion
| ChangePParams     | A change to one or more updatable protocol parameters, excluding changes to major or minor protocol versions ("hard forks") |
```

5. The **Hardfork-combinator** requires to "know" about a potential hardfork 6k/f slots before the epoch change. To meet this requiterement, if the _votingStakeDistribution_ is not finished by the time we reach that point, the computation is forced to complete.
6. On the first `TICK` that occures within the **last 6k/f** slots of the epoch (2 stability windows), the system _solidifies_ the next epoch protocol parameters, see [_solidifyNextEpochPParams_](https://github.com/IntersectMBO/cardano-ledger/blob/a00b723cd93658e67d21f40c771fad80c463d9c4/eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Tick.hs#L180C1-L195C6). The RATIFY and ENACT rules are executed at this point to deterimne the _enactState_. If a governance action has enough votes (meets the thresholds for its type), it is said to be **Ratified** and it is added to the _enactState_.
7. At the next epoch boundary, the system **applies** the _enactState_.
8. A governance action **Expires** if it does not accumulate enough votes during it's lifetime. Its last chance to be ratified is when the _DRepPulser_ talies the votes on the epoch following its expiration. 

> 
> 
> 
> 
> 
> 
> 
>  

Reviewer 2: ```suggestion
```
